screens/Profile/ExternalProfileScreen/ExternalProfileScreen.tsx

```diff
@@ -1,10 +1,104 @@
-import { View, Text } from "react-native"
+import { View, Text, TouchableOpacity } from "react-native"
+import { styles } from "./styles"
+import { globalStyles } from "../../../assets/global/globalStyles"
+import GeneralProfile from "../../../components/GeneralProfile/GeneralProfile"
+import ExpandableDescription from "../../../components/ExpandableDescription/ExpandableDescription"
+import SectionTabs from "../../../components/SectionTabs/SectionTabs"
+import { useState } from "react"
+import { profileStyles } from "../profileStyles"
+import { ProfileEvents } from "../ProfileEvents/ProfileEvents"
+import { ProfileInterests } from "../ProfileInterests/ProfileInterests"
+import { ProfileNetwork } from "../ProfileNetwork/ProfileNetwork"
 
+type Contact = {
+  uid: string
+  firstName: string
+  lastName: string
+  profilePictureUrl?: string
+  description: string
+  interests: string[]
+  qualifications: string[],
+  location: string
+}
+
+const dummyProfile: Contact = {
+  uid: "4",
+  firstName: "Hervé",
+  lastName: "DelaStrite",
+  profilePictureUrl: "",
+  description: "Description, description, descrc hdsjklaf hel fhj fdj bjhd vbfdhj vbdjhl vdjlh vdhj vfdjh hvdu vdh vdfj vdfs hfdj fdj vdfjl d grei hrj heiua fjispd jis gjdrkl gjreks reksl grei gril nrsj njiption, loooooooooong descriiiiiiiiiiption, this is a veeeeeeeeeeeeeeeeeeeeeery loooooooooooooooooooooooooooooooooooooooooooong descriiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiption alala",
```

Comment:
I would advise using _Lorem Ipsum_ for this kind of dummy data as it will make visual checks clearer and more relevant.

[See here for Lorem Ipsum generator](https://loremipsum.io/)